higgstautau/leplep/filters.py

```diff
@@ -0,0 +1,481 @@
+from rootpy.tree.filtering import EventFilter
```

Comment:
@vincecr0ft the filters look good to me. Same comments as for the lephad files on the writing style: add spaces were needed